bears/general/TextLintBear.py

```diff
@@ -0,0 +1,194 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='textlint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+)[\s*|✓]*(?P<severity>'
```

Comment:
Pasted [this](https://github.com/coala/coala-bears/pull/1597#discussion_r122217556) in the wrong comment box, should be pasted here.